app/metrics.yaml

```diff
@@ -1856,3 +1856,53 @@ installation:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+addons:
+  open_addons_in_settings:
+    type: event
+    description: >
+      A user accessed "Add-ons" from the Settings
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/6174
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/8318
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  open_addon_in_toolbar_menu:
+    type: event
+    description: >
+      A user interacted with an installed add-on in the toolbar menu
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/6174
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/8318
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  installed_addons_count:
```

Comment:
Beware that you wouldn't be able to record a value of `0` for a metric type `counter`. Meaning that to record the fact that user has no installed addons, you'd also need to add an explicit `boolean` metric.

Same goes for `enabled_addons_count` below.

Ideally this should use https://mozilla.github.io/glean/book/user/metrics/quantity.html, but that's currently restricted to Gecko only...

Reply:
I changed all of the counters into booleans since we only have 1 add-on right now. In the future, I hope we can get to sending the add-on IDs as seen in https://github.com/mozilla-mobile/fenix/commit/ef84b8b161c4da4b29ab275b1ed25d2214fa35ac.